perses/rjmc/geometry.py

```diff
@@ -551,8 +540,11 @@ def _torsion_potential(self, torsion, phi, beta):
         gamma = torsion.type.phase
```

Comment:
The `phase` should be converted to degrees:
https://github.com/ParmEd/ParmEd/blob/master/parmed/structure.py#L2081

Maybe also call it `phase` instead of `gamma`?